fullstack/package.json

```diff
@@ -11,7 +11,8 @@
     "@ulixee/hero": "1.5.4",
     "@ulixee/hero-core": "1.5.4",
     "@ulixee/hero-interfaces": "1.5.4",
-    "@ulixee/execute-js-plugin": "1.5.4"
+    "@ulixee/execute-js-plugin": "1.5.4",
+    "awaited-dom": "1.3.1"
```

Comment:
Do we want this to be a direct dependency? If we're only pulling types, you can leave it as a devDependency

Reply:
I was getting squiggly lines in Webstorm telling me I needed to make it a dependency. I can move it back.

Reply:
Fixed